BotRoyaleUnity/Assets/Scenes/SpawnFromNetworkTest.unity

```diff
@@ -96,7 +96,6 @@ LightmapSettings:
     m_PVRFilteringAtrousPositionSigmaAO: 1
```

Comment:
The NetworkHandler gameobject in this scene has been disabled, making the spawning not work. We should re-enable it so we have a scene where the network functionality works